src/book/seed-bank/pubmed-github-repositories/almanack_checks.ipynb

```diff
@@ -0,0 +1,245 @@
+      "4         https://github.com/theislab/Scanpy  /private/var/folders/wl/tr6p4_313g5f1hh3lwbxmy...          3636              625  [2017-01-29, 2025-09-04]                      3141              1.157593  2025-09-04T21:59:02.200385Z  0.0.0.post224.dev0+d5e7a3b                  None                 None     None                      None                  True                        True                          False                   True             True                            True                       True                       174                                  27                                      16               95                          9                              4                    NaN             None                     NaN                       None               NaN                     NaN                     None                               0                             0                        None                                  0                  None                      None                    None                                               None                                  0.76                             100.0                           76.0                            24.0                       None                           None                                              None                           None                              None               0.010850        0.545455                         6                          11  {\"almanack-score-numerator\": 6, \"almanack-scor...                            []                               []            11              6   54.545455\n"
+     ]
+    }
+   ],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "df = pd.read_parquet(\"repository_almanack_results_batch_1.parquet\")\n",
+    "\n",
+    "# Show all columns for the first few rows\n",
+    "with pd.option_context(\"display.max_columns\", None, \"display.width\", 2000):\n",
+    "    print(df.head(10))\n"
```

Comment:
Consider taking direct advantage of the HTML display of Pandas dataframes by removing the `print` here.